### ISSUE TITLE
Images no longer have a maximum height

### ISSUE THREAD

**Issue by durdevic** [jossmac/react-images#373]
**Steps to reproduce the behavior:**
Just put this image in the frame, you should see the problem.

Original image:
https://scontent-frx5-1.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/95398599_1114769665571470_2586615852323220060_n.jpg?_nc_ht=scontent-frx5-1.cdninstagram.com&_nc_cat=1&_nc_ohc=lmuRFZql9XIAX8AQKk9&oh=3150beae8cd48b453dd8fce0ba57d7ad&oe=5ED2E9D9

My example: 
(you can see where the image ends by the "close button" in the top right and the carousel numbers on the bottom right)
![Screenshot at May 04 17-41-17](https://user-images.githubusercontent.com/2753836/80984743-8c32ca00-8e2e-11ea-91a4-1d84bd305d37.png)

**Expected behavior:**
To be bound to the frame

**Comment by durdevic** [comment]
@davwheat Sry bro, but this "should be" not a complicated fix. 

Thanks!

**Comment by aperkaz** [comment]
@durdevic if you need that feature as I did, I found out that `react-image-lightbox` works pretty well. Preload images too, and is 50K instead of 155K size. 
https://github.com/frontend-collective/react-image-lightbox

**Comment by aperkaz** [comment]
After tryin to fix this issue for a couple of ours, I gave up and swapped `react-images` with [`simple-react-lightbox`](https://github.com/michelecocuccio/simple-react-lightbox). It has a clean API ahd is higly customizable.

**Comment by BrendanHart** [comment]
For my use case, using `maxHeight: "100vh"` on the `Img` tag inside `src/components/View.js` seems to fix this. If this is desirable, I could submit a pull request.

**Comment by BrendanHart** [comment]
For those looking for an immediate fix without changes to the library, the following CSS seems to apply the same effect:
```
.react-images__view-image {
    max-height: 100vh !important;
}
```

**Comment by ch3ll0v3k** [comment]
> For those looking for an immediate fix without changes to the library, the following CSS seems to apply the same effect:
> 
> ```
> .react-images__view-image {
>     max-height: 100vh !important;
> }
> ```

Dude, you are the best ))) thx

**Comment by douglasrcjames** [comment]
Thanks @BrendanHart ! I found that `max-height: 90vh` worked better as it didn't cut off my caption.

**Comment by froleo** [comment]
Overriding the Carousel style worked for me.

`<Carousel
              styles={{ view: (base, state) => ({ ...base, height: "90vh" }) }}
            />`
            
Also version 1.1.6 does not seem to have this issue

**Comment by davwheat** [comment]
Fixed with 1.2.0-beta.5
